src/wasm-type.h

```diff
@@ -671,6 +671,8 @@ struct TypeBuilder {
     ForwardChildReference,
     // A continuation reference that does not refer to a function type.
     InvalidFuncType,
+    // A non-shared field of a shared heap type.
+    InvalidFieldType,
```

Comment:
This could perhaps be more specific? How about `UnsharedFieldOfShared`?

Reply:
I went with `InvalidUnsharedField`. I think that's more similar to existing error names.